python/test/t_GaussianProcess_std.py

```diff
@@ -58,3 +58,21 @@
 model = ot.GaussianProcess(cov, myTimeGrid)
 print('model=', model)
 print('marginal=', model.getMarginal([0, 2]))
+
+# FIX #2121
```

Comment:
Ideally, this test should also be ported to C++. The equivalent C++ code should not be too different.

Reply:
Yes. I will do it if I have time...